packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -55,12 +55,18 @@ const AccountOption = ({ account, idx }: TAccountsList) => {
             </div>
 
             <span className='account-transfer-form__balance'>
-                <Money
-                    amount={account.balance}
-                    currency={account.currency}
-                    has_sign={Boolean(account.balance && Number(account.balance) < 0)}
-                    show_currency
-                />
+                {is_pending && is_selected_from ? (
```

Comment:
Better to replace `is_pending` variable with a more descriptive word. Not sure what exactly is pending :)

Reply:
true, was thinking if i should make it more specific for this case, which I should since this should show for only pending POA/POI mf clients 👍